includes/Core/Util/Debug_Data.php

```diff
@@ -0,0 +1,272 @@
+					$field['value'] = $none;
+					$field['debug'] = 'none';
+				}
+
+				return $field;
+			},
+			array_merge( $fields, $this->get_module_fields() )
+		);
+	}
+
+	/**
+	 * Gets the field definition for the amp_mode field.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_amp_mode_field() {
+		$mode     = $this->context->get_amp_mode();
+		$mode_map = array(
+			'primary'   => __( 'Primary', 'google-site-kit' ),
+			'secondary' => __( 'Secondary', 'google-site-kit' ),
+		);
+
+		return array(
+			'label' => __( 'AMP Mode', 'google-site-kit' ),
+			'value' => isset( $mode_map[ $mode ] ) ? $mode_map[ $mode ] : __( 'No', 'google-site-kit' ),
+			'debug' => $mode,
+		);
+	}
+
+	/**
+	 * Gets the field definition for the site_status field.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_site_status_field() {
+		$credentials  = new Credentials( new Encrypted_Options( new Options( $this->context ) ) );
+		$is_connected = $credentials->has();
+		$using_proxy  = ( new Authentication( $this->context ) )->get_oauth_client()->using_proxy();
+		$status_map   = array(
+			'connected-site'  => __( 'Connected through site credentials', 'google-site-kit' ),
+			'connected-oauth' => __( 'Connected through OAuth client credentials', 'google-site-kit' ),
+			'not-connected'   => __( 'Not connected', 'google-site-kit' ),
+		);
+
+		if ( $is_connected && $using_proxy ) {
+			$status = 'connected-site';
+		} elseif ( $is_connected && ! $using_proxy ) {
+			$status = 'connected-oauth';
+		} else {
+			$status = 'not-connected';
+		}
+
+		return array(
+			'label' => __( 'Site Status', 'google-site-kit' ),
+			'value' => $status_map[ $status ],
+			'debug' => $status,
+		);
+	}
+
+	/**
+	 * Gets the field definition for the user_status field.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_user_status_field() {
+		$user_options      = new User_Options( $this->context );
+		$has_auth_token    = (bool) $user_options->get( OAuth_Client::OPTION_ACCESS_TOKEN );
+		$has_refresh_token = (bool) $user_options->get( OAuth_Client::OPTION_REFRESH_TOKEN );
+		$is_connected      = $has_auth_token && $has_refresh_token;
+
+		return array(
+			'label' => __( 'User Status', 'google-site-kit' ),
+			'value' => $is_connected
+				? __( 'Authenticated', 'google-site-kit' )
+				: __( 'Not authenticated', 'google-site-kit' ),
+			'debug' => $is_connected ? 'authenticated' : 'not authenticated',
+		);
+	}
+
+	/**
+	 * Gets the field definition for the active_modules field.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_active_modules_field() {
+		$module_slugs = wp_list_pluck( $this->modules->get_active_modules(), 'slug' );
+
+		return array(
+			'label' => __( 'Active Modules', 'google-site-kit' ),
+			'value' => join(
+				/* translators: used between list items, there is a space after the comma. */
+				__( ', ', 'google-site-kit' ),
+				$module_slugs
```

Comment:
For this user-facing value we may want to use the `name` value of the module instead.

Reply:
I thought about that. I suppose we should use names here 👍 Will update.